### DISCUSSION TITLE
Audio processing script for speaker diarisation

### DISCUSSION THREAD

**Discussion by Bodawen** [speechbrain/speechbrain#1633] [Q&A]
I am trying to create a speaker diarisation system using speechbrain. Maybe I'm missing something, but it seems that only the code for experiment is provided in project. So I was wondering if there is a script to show how to implement audio processing speech diarisation using speechbrain's model. i.e. input is a wav audio and output is who spoke when?

**Comment by nauman-daw** [comment]
Hi,
Currently, full pipeline is not supported since we need to insert VAD in a pipeline. However, if you have speech/non-speech boundaries marked (or use speechbrain VAD to get them), you can probably edit data_prep accordingly to achieve your goal. Adding a full pipeline to speechbrain may take some time. Please feel free to open a PR if you plan to do so. 
thanks
